subsys/nrf_security/src/drivers/cracen/cracenpsa/src/key_management.c

```diff
@@ -689,12 +697,16 @@ static psa_status_t export_ecc_public_key_from_keypair(const psa_key_attributes_
 				priv_key.key.ed448 = (struct sx_ed448_v *)key_buffer;
 				pub_key.key.ed448 = (struct sx_ed448_pt *)data;
 			}
+#endif
 			break;
 		default:
 			return PSA_ERROR_NOT_SUPPORTED;
 		}
 	}
-
+#if CONFIG_PSA_NEED_NO_SI_CRYPTO_ED25519
+	si_status = create_ed25519_pubkey(key_buffer, data);
+	*data_length = 32;
```

Comment:
Should probably replace this with either a) A define b) Adding it to the function and having it set there